migration/20160517-remove-edition-is-primary-for-work.sql

```diff
@@ -0,0 +1,223 @@
+ALTER TABLE works ADD COLUMN presentation_edition_id integer;
```

Comment:
I think you also need to explicitly add the index on Work.presentation_edition_id -- it won't be created automatically unless you create the database from scratch.